include/sea_dsa/Graph.hh

```diff
@@ -443,7 +443,7 @@ private:
 
 public:
   typedef Graph::Set Set;
-  typedef boost::container::flat_map<unsigned, Set> accessed_types_type;
+  typedef llvm::DenseMap<unsigned, Set> accessed_types_type;
```

Comment:
this automagically shaved-off ~10% of running time in addAccessedTypes.

Reply:
Hmm... flag for more investigation. Either we are not using `flat_map` correctly, or `accessed_types_type` is fairly big